src/Microsoft.Identity.Client/WsTrust/UsernamePasswordInput.cs

```diff
@@ -25,7 +25,7 @@
 //
 //------------------------------------------------------------------------------
 
-#if DESKTOP || ANDROID || iOS
+#if DESKTOP || ANDROID || iOS || MAC
```

Comment:
it's definitely time for feature flags instead of having to make these changes in every location.

Reply:
not in this PR of course, but we should do it soon with the 3.x work.